app/controllers/lists_controller.rb

```diff
@@ -39,19 +34,28 @@ def new
   def create
     @list = List.new(list_params)
     @list.user = current_user
-    if @list.save
-      flash[:success] = "List successfully Added."
-      # render :json => @list
-      redirect_to lists_path
-    else
-      flash[:warning] = @list.errors.full_messages.join(', ')
+    respond_to do |format|
+      if @list.save
+        format.json { render json: @list, status: :created}
+        flash[:success] = "List successfully Added."
+      else
+        format.json { render json: @list.errors, status: :unprocessable_entity }
+        flash[:warning] = @list.errors.full_messages.join(', ')
+      end
     end
   end
 
+  def destroy
+    @user = current_user
+    @product = Product.find(params[:product_id])
+    Review.find(params[:id]).destroy
+    flash[:success] = 'Review Deleted'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.